.gitlab/ci/sample_code.gitlab-ci.yml

```diff
@@ -121,8 +121,8 @@ homepage_sample_code:
     ### https://docs.meltano.com/guide/orchestration
 
     # Schedule pipelines
-    - meltano schedule gitlab-to-jsonl tap-gitlab target-jsonl @hourly
-    - meltano schedule gitlab-to-postgres tap-gitlab target-postgres @daily --transform=run
+    - meltano schedule gitlab-to-jsonl --extractor tap-gitlab --loader target-jsonl --interval="@hourly"
+    - meltano schedule gitlab-to-postgres --extractor tap-gitlab --loader target-postgres --interval="@daily" --transform=run
```

Comment:
Yea probably should since that's what I switched the docs too after the first review pass. The o.g. spec explicitly called out that add should be default behavior for schedule so I didn't update the use every where and just updated to use the appropriate flags.